rhoas-cli-service-registry/README.adoc

```diff
@@ -243,40 +265,34 @@ Artifacts might include, for example, schemas that define the structure of Kafka
 [NOTE]
 ====
 You can use `rhoas service-registry use` to switch to a specific {registry} instance.
-
-.Selecting a {registry} instance to use
-[source,shell,subs="attributes"]
-----
-$ rhoas service-registry use --name=my-registry
-{registry} instance "my-registry" has been set as the current instance.
-----
 ====
 
 .Procedure
 
 . Upload a {registry} artifact.
 +
 --
-This example uploads a {registry} artifact called `my-artifact` to the {registry} instance.
+This example uploads {registry} artifact content called `my-artifact` to the {registry} instance.
```

Comment:
Line 260/275:  I’m not sure if the proposed change is valid, need to check.

Reply:
I changed this back to the original. The command help refers to artifact "content" so that's where I got it from. I was also trying to make the procedure flow a bit more logically - it seems strange to first upload an artifact and then create the artifact :-).